codelists/models.py

```diff
@@ -562,12 +563,30 @@ def table_with_fixed_headers(self):
         # Identify the index for the two columns we want
         code_header_index = header_row.index(code_header)
         term_header_index = header_row.index(term_header) if term_header else None
+
+        table_rows = self.table[1:]
+        if include_mapped_vmps and self.coding_system_id == "dmd":
+            term_header_index = term_header_index or len(self.table[0])
+            # ignore include_mapped_vmps if coding system is anything other than dmd
+            codes = [row[code_header_index] for row in table_rows]
+            # add in mapped VMP codes
+            previous_vmps_for_this_codelist = vmp_ids_to_previous(codes)
+            for current_vmp, previous_vmp in previous_vmps_for_this_codelist:
+                assert current_vmp in codes
```

Comment:
Surely we want the mapped codes where the previous code is in the list but not the current one (i.e. the dm+d has been updated since this list's creation) ?

Reply:
Yeah, to replicate the cohort-extractor behaviour we need to back to the future